RobotInterrogation/appsettings.json

```diff
@@ -41,373 +41,390 @@
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "Animals",
             "traits": [
-              "Mention an animal in response to 3 different questions.",
-              "Make an animal noise. (If this completes your obsession, wait 30 seconds before killing the investigator.)",
+              "Mention an animal in response to 3 different questions",
+              "Make an animal noise",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "Thoroughness",
             "traits": [
-              "3 times, interrupt the investigator to add detail to a description.",
-              "Continue to describe something until interrupted.",
+              "3 times, interrupt the Investigator to add detail to a description",
+              "Continue to describe something until interrupted",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "Conflict",
             "traits": [
-              "Describe 3 verbal disagreements you've had.",
-              "Insult the investigator 2 times.",
+              "Describe 3 verbal disagreements you've had",
+              "Insult the Investigator, 2 times",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "PatientRobot",
+            "fault": "Long Term Memory",
             "traits": [
-              "You may not mention any people other than strangers or enemies."
+              "You may not discuss anything that happened before you woke up this morning. (You may discuss habits and routines, if they are in the present tense.)"
             ]
           },
           {
             "type": "PatientRobot",
+            "fault": "Friendship",
             "traits": [
-              "You may not discuss anthing that happened before you woke up this morning."
+              "You may not mention any people besides strangers or enemies."
             ]
           },
           {
             "type": "PatientRobot",
+            "fault": "Evaluation",
             "traits": [
-              "You may not express opinions.",
-              "You may only state observable facts."
+              "You may not express opinions. You may only state observable facts."
```

Comment:
I formatted this as a single trait like on the game card, we can split this and other spots with compound things if you prefer it.

Reply:
Seems sensible